internal/api/v1/stage.go

```diff
@@ -232,7 +249,7 @@ func createCertificate(ctx context.Context, cfg *rest.Config, app models.App, sy
 				"kind" : "ClusterIssuer"
 			}
 		}
-        }`, app.Name, app.Org, app.Name, systemDomain, app.Name, app.Name, systemDomain, issuer)
+        }`, app.Name, app.Org, cn, app.Name, app.Name, systemDomain, issuer)
```

Comment:
One more variable to that format string and we should turn it into a go template ;)

Reply:
Well, I removed a variable ;) But yes, the counting is quite onerous.